### ISSUE TITLE
The result is extremly bad

### ISSUE THREAD

**Issue by MonolithFoundation** [joonaskalda/PixIT#8]
None of these samples can be separated..

[归档.zip](https://github.com/user-attachments/files/18123932/default.zip)

Could u please take a look?

The code used:

```
from pyannote.audio import Pipeline
    from pyannote.audio import Model

    pipeline = Pipeline.from_pretrained(
        checkpoint_path="pyannote/speech-separation-ami-1.0",
        use_auth_token="",
        cache_dir="checkpoints/pyannote",
    )

    # run the pipeline on an audio file
    # audio_f = "data/audios/overlap/54.8_59.6.wav"
    # audio_f = "data/audios/overlap/67.0_70.5.wav"
    audio_f = "data/audios/overlap/75.8_79.3.wav"
    audio_f = normalize_audio_to_target(audio_f)
    diarization, sources = pipeline(audio_f)

    # dump the diarization output to disk using RTTM format
    print(diarization)

    # dump sources to disk as SPEAKER_XX.wav files
    import scipy.io.wavfile

    for s, speaker in enumerate(diarization.labels()):
        scipy.io.wavfile.write(
            os.path.join(os.path.dirname(audio_f), f"{speaker}.wav"),
            16000,
            sources.data[:, s],
        )

```

the separation result almost like random gues....

**Comment by joonaskalda** [comment]
Indeed the pre-trained checkpoint (pyannote/speech-separation-ami-1.0) that we provide does not perform well on the samples that you provided. Our training data is the AMI English meeting corpus which is from a very different domain than your TV broadcast data. I recommend either fine-tuning the model or training your own one from scratch. Detailed instructions on how to do this in the pyannote framework are provided in these tutorial notebooks (database setup is exactly the same for PixIT and speaker diarization): 
- https://github.com/pyannote/pyannote-audio/blob/develop/tutorials/adapting_pretrained_pipeline.ipynb
- https://github.com/pyannote/pyannote-audio/blob/develop/tutorials/training_a_model.ipynb

**Comment by MonolithFoundation** [comment]
Hi. have u guys considered opensource some model that trained on a even more diversity data.

As a relatively foundation model, training on more large scale data could make this model more powerful and useful.

**Comment by joonaskalda** [comment]
Yes, definitely on the to-do list. Unfortunately, training this has a high computational cost.

**Comment by MonolithFoundation** [comment]
@joonaskalda how much data needed (in Gigabytes) and how much GPU hours needed for an A100? I could help to contribute.

**Comment by joonaskalda** [comment]
@MonolithFoundation the contribution would be much appreciated! For larger-scale training, the logical step would be to use all openly available diarization datasets (like is done in e.g. [this work](https://www.isca-archive.org/interspeech_2024/baroudi24_interspeech.pdf)). This is around 1000h of single-channel 16kHz data which is about 100Gb of data. The current checkpoint is trained on the AMI-SDM dataset only (100h) with one A100-80gb for 3 days so on the larger-scale dataset an estimate would be 30 days.
